### PR TITLE
feat: add new tabs ui component, add restrictions step to creation form, remove confetti

### PR DESCRIPTION
**Pull request by guerrap**

### CODE REVIEW

File: packages/frontend/src/components/create-campaign/form/restrictions-step/csv-addresses-import/index.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        if (addresses.some((address) => !isAddress(address))) {
+            toast.custom((toastId) => (
+                <ImportFail
+                    toastId={toastId}
+                    message={t("notification.fail.malformed")}
+                />
+            ));
+            return;
+        }
+        if (new Set(addresses).size !== addresses.length) {
```
Comment (luzzif) [inline packages/frontend/src/components/create-campaign/form/restrictions-step/csv-addresses-import/index.tsx:58]: I don't think we necessarily need to fail here. Can we just remove the duplicates? Wdyt?

File: packages/frontend/src/components/create-campaign/form/restrictions-step/index.tsx [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+        if (!prevRestrictions) return true;
+
+        return (
+            prevRestrictions.list.length !== addresses.length ||
+            prevRestrictions.type !== type
+        );
+    }, [addresses, prevRestrictions, type]);
+
+    // this hooks is used to disable and close the step when
+    // the kpi specification gets disabled, after the campaign creation
```
Comment (luzzif) [inline packages/frontend/src/components/create-campaign/form/restrictions-step/index.tsx:79]: Is this comment still applicable? Or copy/paste issue?

**Review by luzzif — CHANGES_REQUESTED** [review]

File: packages/frontend/src/components/create-campaign/form/restrictions-step/csv-addresses-import/index.tsx (hunk 1 above)
Comment (guerrap) [reply packages/frontend/src/components/create-campaign/form/restrictions-step/csv-addresses-import/index.tsx:58]: The reasoning behind failing in this case was to make sure the user knows there are duplicates in the file, not sure, we can also remove the duplicates.

File: packages/frontend/src/components/create-campaign/form/restrictions-step/index.tsx (hunk 2 above)
Comment (guerrap) [reply packages/frontend/src/components/create-campaign/form/restrictions-step/index.tsx]: Oh, yeah this is still relevant, just fixed the comment 76948d6854810025077649b653ec1f19c908b1ba.

File: packages/frontend/src/components/create-campaign/form/restrictions-step/csv-addresses-import/index.tsx (hunk 1 above)
Comment (guerrap) [reply packages/frontend/src/components/create-campaign/form/restrictions-step/csv-addresses-import/index.tsx]: Addressed in a3f83c7fb8b545ebf03794983fefa21a70f5d482.

**Pull request merged**